### ISSUE TITLE
z-index select field

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#7950]
### Bug description

I'm using a select field set to `Select dropdown` and there are some layering issues:
<img width="495" alt="Screenshot 2023-04-19 at 14 04 25" src="https://user-images.githubusercontent.com/69107412/233069661-de94528c-143b-442b-88a5-3fbdbea7f465.png">

### How to reproduce

Add a select field followed by a section, or possibly also other fields.

### Environment

```yaml
Environment
Application Name: Stichting Lutje Geluk
Laravel Version: 10.8.0
PHP Version: 8.2.3
Composer Version: 2.2.7
Environment: local
Debug Mode: ENABLED
URL: lutjegeluk.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 6
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: dev-master PRO

Statamic Addons
doublethreedigital/guest-entries: 2.1.2
statamic/eloquent-driver: 1.2.0
studio1902/statamic-peak-browser-appearance: 2.0-beta.2
studio1902/statamic-peak-commands: 2.0-beta.3
studio1902/statamic-peak-seo: 2.0-beta.2
studio1902/statamic-peak-tools: 3.0-beta.2
```

### Installation

Starter Kit using via CLI

### Antlers Parser

runtime (new)

**Comment by robdekort** [comment]
Added detail: This also happens with the typeahead option.

**Comment by jasonvarga** [comment]
Is that a `select` field like you said? Or is it an `entries` field?

**Comment by robdekort** [comment]
Sorry entries set to select mode

**Comment by jasonvarga** [comment]
Fixed in #7973
